aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/TestScripts/doc_name.ps1

```diff
@@ -0,0 +1,35 @@
+# Yields error Document with name 'v2' not found.
```

Comment:
@martincostello  using NuGet pkg 9.0.0-rc.1.24412.15
` <OpenApiGenerateDocumentsOptions>--document-name v2</OpenApiGenerateDocumentsOptions>`
I'm still getting an error error Document with name 'v2' not found

Should I file an issue on this at https://github.com/dotnet/aspnetcore/issues ?

Reply:
Possibly - I haven't used this option myself, as I only have one OpenAPI document in my app.

Reply:
This is happening because the sample application is only configured with one OpenAPI document, the default one that is added when you call `AddOpenApi` which happens to have the name `v2`. See https://github.com/dotnet/AspNetCore.Docs/commit/8d7b0464dbb1b483430288d39ed37615a4bc4784 with a diff.